test/integration/serverless/run.sh

```diff
@@ -253,6 +324,10 @@ for function_name in "${all_functions[@]}"; do
             echo
             echo "$diff_output"
             echo
+            echo "Raw output:"
+            echo
+            echo "$raw_logs"
```

Comment:
I'm worried this will gum up the output with too much information. Mind if we just print the location of the raw logs instead? If running in CI, we could always tell people where to look for the artifact in the UI.

Reply:
Yeah... this was mostly useful for my own debugging, actually...